### DISCUSSION TITLE
Onboarding PR Configuration Summary - Not reflecting config:base overrides

### DISCUSSION THREAD

**Discussion by setchy** [renovatebot/renovate#9237] [Request Help] [answered]
Hi all

BitBucket Cloud self-hosting user here.

I am using the `config:base` defaults and have overriden the `prConcurrentLimit` and `branchConcurrentLimit` to reduce overloading our CI/CD pipelines

```
  onboardingConfig: {
    extends: ["config:base"],
    prConcurrentLimit: 2,
    branchConcurrentLimit: 2,
  }
```

I notice in the **Configuration Summary** section of the Onboarding PR that it still references the following:

>  Rate limit PR creation to a maximum of two per hour
>  Limit to maximum 20 open PRs at any time

Is this to be expected?  I was anticipating it to mention the concurrent limits defined

**Comment by rarkins** [comment, marked as answer]
It only takes into account the extends array and concatenates all descriptions it finds. It's expected behaviour but not something I'm proud of. It would be challenging to get great because deriving human-understandable descriptions from raw config would be like trying to generate human readable descriptions from CSS or an Apache config, etc. 

Best you comment do is create your own alternative to config:base and make that your extended config. Then it would gather every nested description within it.
